roles/openshift-labels/README.md

```diff
@@ -14,3 +14,4 @@ tasks:
 ```
 When the `target_namespace` is ommitted, the current oc client namespace is used.
 
+This role can apply multiple labels to an object at the same time. This can be done by providing a space seperated list of labels.
```

Comment:
@tylerauerbeck good call-out. Could it be good to maybe add some examples in this README?

Reply:
Added a single and multi-label example. Let me know if you'd like something more specific than what I've got in there.

Reply:
Looks great